browser/src/Services/QuickOpen.ts

```diff
@@ -33,6 +33,13 @@ export class QuickOpen {
             } else {
                 neovimInstance.command("vsp! " + fullPath)
             }
+
+            // TODO If we export the icon, change from arg.icon to whatever
+            // Or have modes
+            if (arg.icon === "chain") {
+                neovimInstance.command("cd! " + arg.detail)
```

Comment:
Oh cool! I just added a `neovimInstance.chdir` method in my PR (#577), but I missed adding the `!` - that's a good call.